test/functional/lua/snippet_spec.lua

```diff
@@ -18,6 +18,20 @@ local retry = t.retry
 describe('vim.snippet', function()
   before_each(function()
     clear()
+    exec_lua(function()
+      local function set_snippet_jump(direction, key)
+        vim.keymap.set({ 'i', 's' }, key, function()
+          if vim.snippet.active({ direction = direction }) then
+            return string.format('<Cmd>lua vim.snippet.jump(%d)<CR>', direction)
+          else
+            return key
+          end
+        end, { desc = 'vim.snippet.jump if active, otherwise ' .. key, expr = true })
```

Comment:
```suggestion
        end, { silent = true, expr = true })
```

For test purposes the description is irrelevant, but I think we should set `silent` and `expr` like the default mapping does.